mods/ra/maps/monster-tank-madness/map.yaml

```diff
@@ -2114,6 +2115,7 @@ Rules:
 		Tooltip:
 			GenericVisibility: Enemy
 			ShowOwnerRow: false
+		AnnounceOnSeen:
```

Comment:
No, it just plays audio/visual notifications triggered by EnemyWatcher and ScriptTrigger will listen to EnemyWatcher and do the script stuff so this is essentially doing nothing here.